src/controller/java/src/matter/controller/MatterController.kt

```diff
@@ -66,12 +66,7 @@ interface MatterController : Closeable, InteractionClient {
     fun onICDRegistrationInfoRequired()
 
     /** Notifies when the registration flow for the ICD completes. */
-    fun onICDRegistrationComplete(
-      errorCode: Int,
-      icdNodeId: Long,
-      icdCounter: Long,
-      icdDeviceInfo: ICDDeviceInfo
-    )
+    fun onICDRegistrationComplete(errorCode: Int, icdDeviceInfo: ICDDeviceInfo)
```

Comment:
We've adapted coroutines to convert all IM APIs into synchronized APIs in Kotlin API. Could we replace the callback mechanism with a synchronous call for consistency in Kotlin APIs?

Reply:
We are in the process to decouple the Kotlin lib with Java lib, so we should not add additional depenedce. Please remove "import chip.devicecontroller.ICDDeviceInfo"

Reply:
As you know, I am also working on a separate implementation of the kotlin library and java library JNI. If I modify it now, I think that most Interface Classes (ICDDeviceInfo etc.) in Java should be implemented identically in Kotlin, and the only difference is the package name. Therefore, the Java package name is now used in kotlin-controller, but I think it would be better to change it while implementing the kotlin class.